docs/components/slider.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebarDepth: 0
+---
+
+# Slider
+
+This is an overview of the most common usage of `<Slider>`.
+
+<DocExampleBox codeBox="https://codesandbox.io/s/8l36nz28p8?module=%2Fsrc%2FApp.vue">
+
+```html
+<Slider :value="80" @valueChange="onValueChanged" />
+```
+
+<SliderDoc />
+</DocExampleBox>
+
+`Slider` provides two-way data binding using v-model.
+
+```html
+<Slider v-model="value" />
+```
+
+## Props
+
+| Name     | Type   | Default | Web | Mobile |
+| -------- | ------ | ------- | --- | ------ |
+| value    | Number | `0`     | ✔   | ✔      |
```

Comment:
description column should be added on both props and events section

Reply:
Added description in d3c44a61e4d5a2c80e231093710514d2ba83e79a